### PR TITLE
WIP: Separate "Other Meetings" by day in its own block, and include links.

### PR DESCRIPTION
**Pull request by FlipperPA**
This works, but could be more DRY. I'm still figuring out ReactJS scope and syntax. In particular, the two methods `getMeetingLink` and `setMeeting` are duplicated in both `meeting.jsx` and `table.jsx`. I've struggled trying to import one from the other file, since the context of `this` is different. Any help would be appreciated!

That said, I think this can be merged to master and refactored down the road, as its an improvement over what we have now and works.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
         "react-map-gl": "^4.0.5",
         "sass": "^1.15.2",
         "sass-loader": "^7.1.0",
-        "viewport-mercator-project": "^6.1.0"
+        "viewport-mercator-project": "^6.1.0",
+        "react-dropdown-tree-select": "^1.20.1"
```
Comment (FlipperPA) [inline package.json]: Wups, this line isn't quite needed yet, but will be soon.

**Comment by FlipperPA** [comment]
@joshreisner I think this is probably ready to merge. I've got the Region Tree work on another branch. We can refactor this to be better later, as I learn more, if that's cool!

**Comment by joshreisner** [comment]
Hey Tim! I think I see in the history that you tried a MeetingLink component, I think that'd be great to do that, since some of the same functionality is repeated in `Table` and `Meeting`.

Also I'm a little wary of the `loop_today` construction because I don't think we're sorting the meetings anywhere, so they might not appear in chrono order. We should probably be sorting in `App`, but I wonder if we could refactor this loop/counter construction with `settings.days` and some filters.

Also in terms of presentation I think we should probably try to separate the `ol`s and hide the numbering.

LMK what you think of these ideas. I can help out with this if you want.

**Comment by joshreisner** [comment]
Ah, I tried to do it but I think I would need to check out your copy of the repo. I moved the `MeetingLink` class out of Table and just called in `Link` but then I ran into version issues.

Wonder if it might make more sense to move the development to branches on this repo?

**Comment by FlipperPA** [comment]
@joshreisner Absolutely, that makes sense! I'll push my branches up to the main repository. I'm just in the habit of forking because I don't usually have push access to the main repo, but in this case, it's just an extra / un-necessary step.

**Comment by FlipperPA** [comment]
Closing this down, since we'll be merging from the branch within this repo.

**Pull request closed without merging**